R/beers-r6.R

```diff
@@ -0,0 +1,153 @@
+Beers_Interpolator <- R6::R6Class("Beers_Interpolator",
```

Comment:
This is a pretty weird name, but I think we can remove R6 entirely here

Reply:
Yep, R6 is gone!